acl-lib/src/cmr/acl/core.clj

```diff
@@ -59,10 +61,13 @@
   to get sids, check if they are stored on the context."
   [context]
   (or (util/get-real-or-lazy context :sids)
-      (let [{:keys [token]} context]
-        (if token
-          (echo-tokens/get-current-sids context token)
-          [:guest]))))
+      (let [{:keys [token]} context
+            sids (if token
+                  (access-control/get-current-sids context token)
+                  [:guest])]
+        (if (string? sids)
```

Comment:
not sure why I was getting different formats here.  I will just return sids as is, and hopefully not break anything :-)